### PR TITLE
ci: Update tokens action

### PR DESCRIPTION
**Pull request by witwash**
#### Summary

- removed reviewers (assigned automatically based on the files changed)
- commit step would be skipped if `tailor.dart` file is not changed

#### Testing steps

Check if the action is working as intended.

#### Follow-up issues

None

#### Check during review

- Verify against Jira issue.
- Is the PR over 300 additions? Consider rejecting it with advice to split it. Is it over 500 additions? It should definitely be rejected.
- Unused code removed.
- Build passing.
- Is it a bug fix? Check that it is covered by a proper test (unit or integration).

### CODE REVIEW

File: .github/workflows/tokens-pr.yml [hunk 1]
```diff
@@ -21,12 +21,18 @@ jobs:
       - name: Generate tokens files
         run: melos gen_theme
 
+      - name: Check if tailor file was changed
+        id: tailor_changed
+        run: git status --porcelain | grep -q 'tailor.dart'
+        continue-on-error: true
+
       - name: Commit changes
+        if: steps.tailor_changed.outcome == 'success'
```
Comment (witwash) [inline .github/workflows/tokens-pr.yml:30]: I think this is the right way, buy maybe you have a better idea 🤔

Comment (ookami-kb) [reply .github/workflows/tokens-pr.yml:30]: I guess you can just use `success()` for checking all the previous steps without specifying id: https://docs.github.com/en/actions/learn-github-actions/expressions#status-check-functions

But shouldn't it also be applied to creating PR step?

Comment (witwash) [reply .github/workflows/tokens-pr.yml:30]: No, this is for the case when the value itself has changed (in `tokens_dark/light.dart`), but the structure didn't (no removes/renames/additions). In that case, the `optimus_tokens.tailor.dart` file would be the same, but we still want the PR with new values.

Comment (ookami-kb) [reply .github/workflows/tokens-pr.yml:30]: Ah, then yeah, makes sense.

**Review by ookami-kb — APPROVED** [review]

File: .github/workflows/tokens-pr.yml (hunk 1 above)
Comment (ookami-kb) [reply .github/workflows/tokens-pr.yml:30]: But cannot you just do `git add optimus/lib/src/theme/optimus_tokens.tailor.dart` always? If it's the same, it just won't be added, no?

**Comment by witwash** [comment]
@ookami-kb,
so, can I merge this one? 🤔

**Review by ookami-kb — APPROVED** [review]

**Pull request merged**